.rubocop.yml

```diff
@@ -34,7 +33,7 @@ Lint/UselessAccessModifier:
 Metrics/AbcSize:
   Max: 20
 Metrics/ClassLength:
-  Max: 240
+  Max: 300
```

Comment:
@pathawks yup, that was something I explicitly asked for, it might have been in another pull request but I do remember explicitly asking for it to change if it was a problem.